src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1117,6 +1117,8 @@ external gteq   ::= collapse GTEQ GT EQ       { name = ">=" }
 external ltlteq ::= collapse LTLTEQ LT LT EQ  { name = "<<=" }
 external ltlt   ::= collapse LTLT LT LT       { name = "<<" }
 external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
+external oror   ::= collapse OROR OR OR       { name = "||" }
+external or     ::= onlySingle OR             { name = "|" }
```

Comment:
`oror.rs` checks for this. It's a stripped down version of some file from `ctrs` which caught this.